core/src/channel_group/mod.rs

```diff
@@ -21,15 +24,29 @@ pub struct ChannelGroup {
     audio_params: AudioStreamParams,
 }
 
+/// Options for initializing a new ChannelGroup.
 pub struct ChannelGroupConfig {
+    /// Channel initialization options (same for all channels).
     pub channel_init_options: ChannelInitOptions,
+
+    /// Amount of VoiceChannel objects to be created.
+    /// (Number of MIDI channels)
```

Comment:
Probably clarify here that the number is generally 16.

Do we have defaults for these? I can't remember

Reply:
Will specify this. In other config structs where we implement `Default` I did write that the default value is 16.

We don't implement `Default` for `ChannelGroupConfig` though, probably because it takes in `AudioStreamParams` which does not have a default. 

Although I could make a default of 48kHz stereo, like I did in the clib. Should I do that?

Reply:
Nah I think defaulting the sample rate is a bit weird. Just clarifying that channel_count is generally going to be 16 for midi is ok

Another approach would be to use a builder pattern for this, taking the audio params in the initializer but then chaining other parameters as needed, letting you make this field have a default value

Reply:
> Another approach would be to use a builder pattern for this, taking the audio params in the initializer but then chaining other parameters as needed, letting you make this field have a default value

Seems pretty unnecessary imo since it is just a struct so the functions would just set the values and do nothing else. So probably just initializing a struct with the values is more straightforward. But I could still add it, I don't mind.